pkg/virtualKubelet/liqoNodeProvider/reconciler.go

```diff
@@ -34,6 +34,21 @@ import (
 	"github.com/liqotech/liqo/pkg/utils/slice"
 )
 
+func (p *LiqoNodeProvider) reconcileNodeFromNode(event watch.Event) error {
+	var node v1.Node
+	unstruct, ok := event.Object.(*unstructured.Unstructured)
+	if !ok {
+		return errors.New("error in casting Node")
+	}
+	if err := runtime.DefaultUnstructuredConverter.FromUnstructured(unstruct.Object, &node); err != nil {
+		klog.Error(err)
+		return err
+	}
```

Comment:
What is the purpose of de-unstructuring the node resource in this case?

Reply:
None I think 😅 I'm removing it